release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1024,6 +1024,15 @@ With this release, Operator authors can configure their Operator projects with s
 
 For more information, see xref:../operators/operator_sdk/osdk-multi-arch-support.adoc#osdk-multi-arch[Configuring Operator projects for multi-platform support].
 
+[id="ocp-4.14-builds"]
+=== Builds
+
+The Source-to-Image (S2I) tool is now generally available in {product-title} {product-version}. With this update, you can use the S2I tool to build container images from source code, and transform application code into ready-to-deploy container images. This feature enhances the platform's ability to support reproducible containerized application development. 
+
+For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_software_collections/2/html/using_red_hat_software_collections_container_images/sti#doc-wrapper[Using Source-to-Image (S2I) tool]. 
+
+BuildCSIVolume is now generally available in {product-title} {product-version}.
```

Comment:
Should BuildCSIVolume be Build CSI Volumes? As that is how you have referenced the feature in the Tech Preview table.